minecraft_servers/byen/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+    "server_name": "byen",
+    "nice_name": "Byen",
+    "direct_ip": "mc.byen.net",
+    "server_wildcards": [
+      "byen.net",
+      "%.byen.net"
+    ],
+    "social": {
+      "web": "https://byen.net",
+      "web_shop": "https://butik.byen.net",
+      "web_support": "https://discord.byen.net",
+      "discord": "https://discord.byen.net"
```

Comment:
As custom discord invite links are only supported for LabyMod partners at the moment.
Please use a discord invite link.

Documentation reference: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url